pom.xml

```diff
@@ -50,8 +50,8 @@
         <apache.httpclient.version>4.5.2</apache.httpclient.version>
         <bouncycastle.bcpkix.version>1.55</bouncycastle.bcpkix.version>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
-        <jersey.version>2.25</jersey.version>
-        <jetty.version>9.2.19.v20160908</jetty.version>
+        <jersey.version>2.26-b06</jersey.version>
+        <jetty.version>9.4.6.v20170531</jetty.version>
```

Comment:
@The-Alchemist Honestly not sure which would be preferable, I have generally tried to avoid multiple builds where possible. If we do a maven classifier, are we going to be able to get a default that will continue to work with jdk7? My biggest concern would be with compatibility -- it's not ideal if the maven coordinates change or downstream projects then also are required to add classifiers.